src/db/redis/client.ts

```diff
@@ -2,18 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { logger } from "@sentry/utils";
+import { logger } from "@sentry/core";
 import { createRedisInstance } from "./util";
-import { createRedisMockInstance } from "./util-mock";
 import type { Redis } from "ioredis";
+import MockRedis from "ioredis-mock";
 
 export const REDIS_ALL_BREACHES_KEY = "breaches";
 export const REDIS_ALL_DATA_BROKERS_KEY = "dataBrokers";
 
 let singleton: Redis;
 export const redisClient = () => {
   if (process.env.REDIS_URL?.includes("redis.mock")) {
-    singleton = createRedisMockInstance();
+    singleton = new MockRedis();
```

Comment:
this was a one-liner in `utils-mock` file, which I moved here for clarity and to eliminate needing another test for that file